src/kbsb/oldsite/api_oldsite.py

```
@@ -0,0 +1,30 @@
+):
+    """
+    call mail_relay
+    """
+    try:
+        if mrv.attachments is None:
+            mrv.attachments = []
+        logger.info(f"{mrv.sender=} {len(mrv.attachments)}")
+        validate_header(apikey)
+        await mail_relay(mrv)
+    except RdException as e:
+        raise HTTPException(status_code=e.status_code, detail=e.description)
```

Ruff (0.15.4)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)